src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="big form"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="small"
+      action="#"
+    >
+      <input
+        class="form small"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'small' does not describe the content or function of the input.